lib/td_core/search/cluster.ex

```diff
@@ -8,9 +8,10 @@ defmodule TdCore.Search.Cluster do
     indexes =
       Enum.reduce(aliases, %{}, fn
         {index, alias_name}, acc ->
-          with %{settings: settings} = config <- Map.fetch!(indexes, index),
-               %{} = config <- Map.put(config, :settings, Map.merge(settings, defaults)) do
-            Map.put(acc, String.to_atom(alias_name), config)
+          with index_keyword = config <- Keyword.fetch!(indexes, index),
```

Comment:
Both variables, index_keyword and config represent the same value

Reply:
removed :-)